HISTORY.md

```diff
@@ -6,6 +6,52 @@ The format is based on [Keep a
 Changelog](https://keepachangelog.com/en/1.0.0/), and this project
 adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased 5.0.0
+
+In version 5.0, we introduce a new [file uploader](https://uploadcare.com/docs/file-uploader/), which is now the default for Django projects. If you prefer to continue using the old jQuery-based widget, you can enable it by setting the `use_legacy_widget` option in your configuration:
+
+```python
+UPLOADCARE = {
+    ...,
+    "use_legacy_widget": True,
+    "legacy_widget": {
+        "version": "3.x",
+    }
+}
+```
+
+It's important to mention that these changes only apply to Django projects, and there are no breaking changes for non-Django projects.
+
+Additionally, please take note that some settings have been renamed in this update (see the next section).
```

Comment:
Maybe put it under the "breaking changes" subtitle?

Reply:
I really think this belongs to the summary.